.github/workflows/build-publish.yaml

```diff
@@ -36,11 +36,20 @@ jobs:
       - name: Setup dependencies
         run: go mod download
       - name: Run tests
+        run: go test -race ./...
+
+  test-internal:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Dispatch workflow
         env:
-          GH_USER: nestobot
           GH_TOKEN: ${{ secrets.TESTING_TOKEN }}
```

Comment:
What permissions are assigned to this token? Could we give it a more descriptive name?

Reply:
Great, could we give the token a more descriptive variable name, please :)